tasks-by-users/AbbasShaikh0304/01-E-Commerce website/index.html

```diff
@@ -0,0 +1,387 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css"
+    />
+    <title>Abbas Shaikh : E-commerce Website</title>
```

Comment:
Generally ,title tags contains website name not author's name,
you can use meta tags to add author details to your files.

@AbbasShaikh0304

Reply:
Thanks for the suggestion. i'll take account of it @Tauqeer-Ahmed-99

Reply:
Fixed it @Tauqeer-Ahmed-99